<launch>
  <!-- Parameters -->
  <arg name="mode" default="0" />
  <arg name="xzplane" default="false" />
  <arg name="frame_id_map" default="map" />
  <arg name="frame_id_odom" default="odom" />
  <arg name="frame_id_base" default="base_link" />
  <arg name="beta_1" default="1.11840023647110e-03" />
  <arg name="beta_2" default="1.08841788897499e-02" />
  <arg name="beta_3" default="4.49500672685259e-07" />
  <arg name="beta_4" default="7.03448116182080e-05" />
  <arg name="beta_5" default="2.88386534626639e-06" />
  <arg name="beta_6" default="2.46471128604309e-05" />

  <!-- Remapping -->
  <arg name="marker" default="marker" />
  <arg name="cmd" default="cmd_vel" />

  <node pkg="tuw_marker_slam" type="tuw_marker_slam_node" name="tuw_marker_slam" output="screen">
      <param name="mode" value="$(arg mode)" />
      <param name="xzplane" value="$(arg xzplane)" />
      <param name="frame_id_map" value="$(arg frame_id_map)" />
      <param name="frame_id_odom" value="$(arg frame_id_odom)" />
      <param name="frame_id_base" value="$(arg frame_id_base)" />
      <param name="beta_1" value="$(arg beta_1)" />
      <param name="beta_2" value="$(arg beta_2)" />
      <param name="beta_3" value="$(arg beta_3)" />
      <param name="beta_4" value="$(arg beta_4)" />
      <param name="beta_5" value="$(arg beta_5)" />
      <param name="beta_6" value="$(arg beta_6)" />
      <remap from="marker" to="$(arg marker)" />
      <remap from="cmd" to="$(arg cmd)" />
  </node>
</launch>
